public/doc/openapi-3-v1.0.json

```diff
@@ -3322,8 +3322,8 @@
             "type": "string"
           },
           "mqtt_client_id": {
-            "example": "The GUID of MQTT client being used",
-            "title": "MqttClientId",
+            "example": "The GUID of the RedHat connect client being used",
+            "title": "RedHatConnectClientId",
```

Comment:
@lgalis Are the other title's ok in the Source object that you can use directly in the UI as defaults?

Reply:
I will add titles where the titles are missing. The info object will be the exception - as there will not be titles for its properties.